tests/unit/models/skill-test.js

```diff
@@ -0,0 +1,15 @@
+import {
+  moduleForModel,
+  test
+} from 'ember-qunit';
+
+moduleForModel('skill', 'Skill', {
+  // Specify the other units that are required for this test.
+  needs: []
+});
+
+test('it exists', function() {
+  var model = this.subject();
+  // var store = this.store();
+  ok(!!model);
```

Comment:
What is a test doing in here?! Actually, that's cool. Wondering what this "ok(!!model)" syntax is all about?